definitions/marts/looker_studio_marts/ecf/ls_ecf_correct_cohort_checker.sqlx

```diff
@@ -0,0 +1,241 @@
+        has_induction_start_date: "TRUE/FALSE if the induction_start_date field is not null."
+    }
+}
+
+/*** ECF Cohort Corrections Looker Studio Data Mart ***/
+
+/*
+  Objective:
+    To gather all ECTs & Mentors inducted within 2021-2023 cohorts for evaluation as to whether they are allocated to correct the cohort based on criteria below.
+
+  Instructions:
+    For ECTs in the "wrong" cohort with declarations:
+      WHERE
+        eligible_for_funding = TRUE
+        AND
+        participant_role = 'ECT'
+        AND 
+        declaration_id IS NOT NULL
+        AND
+        niot_partnership_chk = 0
+        AND
+        start_date_suggestion_match = 0
+      
+    For Mentors in the "wrong" cohort with declarations:
+      WHERE
+        eligible_for_funding = TRUE
+        AND
+        participant_role = 'Mentor'
+        AND 
+        declaration_id IS NOT NULL
+        AND
+        niot_partnership_chk = 0
+        AND
+        earliest_dec_suggestion_match = 0
+
+    For ECTs in the "wrong" cohort without declarations:
+      WHERE
+        eligible_for_funding = TRUE
+        AND
+        participant_role = 'ECT'
+        AND 
+        declaration_id IS NULL
+        AND
+        has_relevant_declarations = FALSE
+        AND
+        niot_partnership_chk = 0
+        AND
+        start_date_suggestion_match = 0
+
+  Dependencies:
+    dataform.ecf_declarations
+*/
+
+-- Gather ECF Declarations
+WITH declarations AS( 
+  SELECT
+     -- Manually assign the user_id for specific participant profile where the profile no longer exists in latest_cpd data and has no superceeded record to call upon.
+     CASE 
+      WHEN dec.participant_profile_id = '8cd0928c-ecad-482f-95cd-6a0ff3e840ad' THEN '79dee4de-a6dd-407c-ab1c-690fcb6d0056'
+      ELSE dec.user_id
+     END AS user_id
+    ,dec.participant_profile_id
+    ,dec.participant_course AS participant_role
+    ,dec.schedule_identifier
+    ,dec.induction_start_date AS induction_start_date
+    ,dec.cohort
+    ,dec.eligible_for_funding
+    ,dec.partnership_id
+    ,dec.partnership_relationship
+    ,dec.lead_provider_name AS induction_lead_provider_name
+    ,dec.cohort_rolled_over_21_24
+    ,IF(dec.state IN ('submitted', 'paid', 'payable', 'eligible'),dec.declaration_id, NULL) AS declaration_id
+    ,IF(dec.state IN ('submitted', 'paid', 'payable', 'eligible'),DATE(declaration_date), NULL) AS declaration_date
+    ,IF(dec.state IN ('submitted', 'paid', 'payable', 'eligible'),dec.declaration_type, NULL) AS declaration_type
+    ,IF(dec.state IN ('submitted', 'paid', 'payable', 'eligible'), state, NULL) AS declaration_state
+    ,IF(dec.state IN ('submitted', 'paid', 'payable', 'eligible'),dec.cpd_lead_provider_name, NULL) AS lead_provider_name
+    ,MIN(CASE WHEN state IN ('paid', 'payable', 'eligible') THEN DATE(declaration_date) ELSE DATE(2050,12,31) END) OVER (PARTITION BY participant_profile_id) AS earliest_declaration_date

```

Comment:
This check already exists as a field it's called funded_declaration

Reply:
This is still in the code? funded_declaration should replace the state in check.